diff --git a/data/meson.build b/data/meson.build
index 7622a6d..f5795db 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -33,7 +33,7 @@ appstream_file = i18n.merge_file(

 appstream_util = find_program('appstream-util', required: false)
 if appstream_util.found()
-  test('Validate appstream file', appstream_util, args: ['validate', appstream_file])
+  test('Validate appstream file', appstream_util, args: ['validate-relax', '--nonet', appstream_file])
 endif

 install_data('com.vysp3r.ProtonPlus.gschema.xml',
@@ -45,4 +45,4 @@ if compile_schemas.found()
   test('Validate schema file',
        compile_schemas,
        args: ['--strict', '--dry-run', meson.current_source_dir()])
-endif
\ No newline at end of file
+endif
