diff --git a/data/com.vysp3r.ProtonPlus.desktop.in b/data/com.vysp3r.ProtonPlus.desktop.in
index 7640e8c..d090278 100644
--- a/data/com.vysp3r.ProtonPlus.desktop.in
+++ b/data/com.vysp3r.ProtonPlus.desktop.in
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=ProtonPlus
-Exec=com.vysp3r.ProtonPlus
+Exec=protonplus
 Icon=com.vysp3r.ProtonPlus
 Terminal=false
 Type=Application
diff --git a/src/meson.build b/src/meson.build
index d834598..5af8488 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -83,7 +83,7 @@ protonplus_deps = [
 ]
 
 executable(
-  meson.project_name(),
+  'protonplus',
   ui_gresource,
   css_gresource,
   icons_gresource,
@@ -92,4 +92,4 @@ executable(
   dependencies: protonplus_deps,
   include_directories: config_inc,
   install: true
-)
\ No newline at end of file
+)
